src/components/posts/PostItem.js

```diff
@@ -28,16 +35,28 @@ function PostItem(props) {
           </div>
           <h5>{props.content}</h5>
         </div>
+        <div className={classes.likeCount}>{likeCount}</div>
```

Comment:
This like count looks a little bit lonely, it isn't easily obvious that this is the number of likes

Reply:
Yeah it's unfinished, was just proof of functionality